# ==============================================================================
# REGRESSOR COMMIT DIFF
# ==============================================================================
# File: testing/web-platform/meta/streams/readable-byte-streams/construct-byob-request.any.js.ini
# Commit: 425cbe3c033b
# Full Hash: 425cbe3c033bfc25795979513ebd2bb9fda7338d
# Author: moz-wptsync-bot <wptsync@mozilla.com>
# Date: 2024-11-28 21:54:11
# Regressor Bug: 1933829
# File Overlap Count: 1
# Description:
#   Bug 1933829 - [wpt-sync] Update web-platform-tests to 2d5d222739fdf91743f8cbc58094976a613194e7, a=testonly
#   
#   MANUAL PUSH: wpt sync bot
#   
#   wpt-head: 2d5d222739fdf91743f8cbc58094976a613194e7
# ==============================================================================

diff -r 732bcd7ac971 -r 425cbe3c033b testing/web-platform/meta/streams/readable-byte-streams/construct-byob-request.any.js.ini
--- a/testing/web-platform/meta/streams/readable-byte-streams/construct-byob-request.any.js.ini	Wed Nov 27 22:05:28 2024 +0000
+++ b/testing/web-platform/meta/streams/readable-byte-streams/construct-byob-request.any.js.ini	Wed Nov 27 22:05:56 2024 +0000
@@ -15,19 +15,22 @@
     ERROR
 
 [construct-byob-request.https.any.shadowrealm-in-serviceworker.html]
-  expected: TIMEOUT
+  expected:
+    if os == "win": ERROR
+    if os == "linux": [TIMEOUT, ERROR]
+    [ERROR, TIMEOUT]
 
 [construct-byob-request.https.any.shadowrealm-in-audioworklet.html]
-  expected: TIMEOUT
+  expected: ERROR
 
 [construct-byob-request.any.shadowrealm-in-dedicatedworker.html]
-  expected: TIMEOUT
+  expected: ERROR
 
 [construct-byob-request.any.shadowrealm-in-window.html]
   expected: ERROR
 
 [construct-byob-request.any.shadowrealm-in-sharedworker.html]
-  expected: TIMEOUT
+  expected: ERROR
 
 [construct-byob-request.any.shadowrealm-in-shadowrealm.html]
   expected: ERROR